orchestrator-salt/src/main/resources/salt/salt/datalake_resize/init.sls

```diff
@@ -0,0 +1,6 @@
+/opt/salt/scripts/check_atlas_updated.sh:
+  file.managed:
+    - makedirs: True
+    - mode: 750
+    - source: salt://datalake_resize/scripts/check_atlas_updated.sh
+    - template: jinja
```

Comment:
I don't see any jinja element in the script, so this seems unnecessary to me

Reply:
I'll remove this.